lib/workers/repository/cache.ts

```diff
@@ -68,19 +70,31 @@ async function generateBranchCache(
         // Do nothing
       }
     }
+    let isBehindBase = false;
+    if (sha) {
+      try {
+        // TODO: fix types (#7154)
+        isBehindBase = await isBranchBehindBase(branchName, baseBranch!);
+      } catch (err) /* istanbul ignore next */ {
+        // Do nothing
```

Comment:
Not even a log? I assume this is not an expected result?

Reply:
Yes a log should be added here but I expect it to never be called.

Reply:
if it should never happen, we shouldn't catch here an let unexpected error throw up